tests/lib/snaps/test-snapd-mount-control/bin/mntctl

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+
+import argparse
+import ctypes
+import ctypes.util
+import os
+import subprocess
+import sys
+
+
+class MountFlags:
+    MS_RDONLY = 1
+    MS_NOSUID = 2
+    MS_NODEV = 4
+    MS_NOEXEC = 8
+    MS_SYNCHRONOUS = 16
+    MS_REMOUNT = 32
+    MS_MANDLOCK = 64
+    MS_DIRSYNC = 128
+    MS_NOATIME = 1024
+    MS_NODIRATIME = 2048
+    MS_BIND = 4096
+    MS_MOVE = 8192
+    MS_REC = 16384
+    MS_SILENT = 32768
+    MS_POSIXACL = (1 << 16)
+    MS_UNBINDABLE = (1 << 17)
+    MS_PRIVATE = (1 << 18)
+    MS_SLAVE = (1 << 19)
+    MS_SHARED = (1 << 20)
+    MS_RELATIME = (1 << 21)
+    MS_KERNMOUNT = (1 << 22)
+    MS_I_VERSION = (1 << 23)
+    MS_STRICTATIME = (1 << 24)
+    MS_LAZYTIME = (1 << 25)
+
+    NAME_TO_VALUE = {
```

Comment:
We IIRC have some of this in `mountinfo` helper, perhaps could be shared.

Reply:
If you are referring to `tests/lib/tools/mountinfo.query`, it doesn't seem to be there, though indeed it would seem like a good place to store these definitions if there was a need to share them.

For the time being, I'm not sure moving them there would be worth, since it would complicate building the test snap (we'd need to import into the snap).